include/aspect/material_model/grain_size.h

```diff
@@ -275,6 +275,23 @@ namespace aspect
         double              mantle_temperature;
         double              surface_temperature;
 
+        /**
+         * Functions and parameters controlling conversion from interface roughness to grain size,
+         * used in pinned state formulation of grain damage.
+         *
+         * Detailed description of this approach can be found in Appendix H.1 of
+         * Bercovici, David, and Yanick Ricard (2012).
+         * Mechanisms for the generation of plate tectonics by two-phase grain-damage
+         * and pinning. Physics of the Earth and Planetary Interiors 202 (2012): 27-55.
+         */
+        std::vector<double> volume_fraction_phase_one;
+
+        double moment_of_grain_size_distribution (const int n) const;
```

Comment:
Can this be an unsigned int or can it be negative?
```suggestion
        double moment_of_grain_size_distribution (const unsigned int n) const;
```